docs/user/settings.rst

```diff
@@ -297,6 +297,60 @@ check when running on multiple servers. Make sure it is always greater
 than the total iperf3 check time, i.e. greater than the TCP + UDP test
 time. By default, it is set to **600 seconds (10 mins)**.
 
+.. _openwisp_monitoring_auto_wifi_client_check:
+
+``OPENWISP_MONITORING_AUTO_WIFI_CLIENT_CHECK``
```

Comment:
I think we should use "clients" (plural) instead of client in all references to this feature.